src/DynamoCore/Models/RunSettings.cs

```diff
@@ -22,7 +22,10 @@ public class RunSettings : NotificationObject
         private bool runTypesEnabled;
         private bool runTypesComboBoxToolTipIsEnabled;
 
-        internal bool ForceAutomaticWithoutRun { get; set; } = false;
+        /// <summary>
+        /// This static property with true value will be used when opening a dyn file with RunMode = Automatic and we don't want to execute the graph
+        /// </summary>
+        internal static bool ForceAutomaticWithoutRun { get; set; } = false;
```

Comment:
I would rename this to `ForceBlockRun` since it blocks all types of run

Reply:
@QilongTang the property was renamed in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12954/commits/bea9cb1cd60eccc2293917eff260dd644f0faead